source/lib/rocprof-sys/library.cpp

```diff
@@ -37,6 +37,7 @@
 #include "core/gpu.hpp"
 #include "core/locking.hpp"
 #include "core/perfetto_fwd.hpp"
+#include "core/rocprofiler-sdk.hpp"
```

Comment:
```suggestion
#include "core/rocprofiler-sdk.hpp"
```
Won't be needed once you move `is_vaapi_tracing_enabled()` to `config.cpp`